optimum_benchmark/backends/base.py

```diff
@@ -49,8 +49,8 @@ class BackendConfig(ABC):
     intra_op_num_threads: Optional[int] = None
 
     # isolation options
-    initial_isolation_check: bool = True
-    continous_isolation_check: bool = True
+    initial_isolation_check: bool = False
+    continous_isolation_check: bool = False
```

Comment:
I think this should be left to True by default. It is an useful feature.

Reply:
okay! I had the impression that it doesn't work with docker containers so I thought it'd make sense to  disable it until it's stable. but it seems it works fine with docker.

Reply:
Ah yes with docker you need to use `docker run --pid host ...` for it to work. I had the same issue, should have documented it

Reply:
https://github.com/fxmarty/transformers-regression-test/commit/9531138bd458fe2259a825e3c7d79a539e2abad6